packages/medusa/src/api/admin/tax-regions/validators.ts

```diff
@@ -57,6 +58,16 @@ export const AdminCreateTaxRegion = z.object({
   metadata: z.record(z.unknown()).nullish(),
 })
 
+export const AdminCreateTaxRegion = WithAdditionalData(
+  CreateTaxRegion,
+  (schema) => {
+    return schema.refine((data) => data.parent_id || data.provider_id, {
```

Comment:
question: Does this work as expected? I think I would have expected this to be `!data.parent_id && !data.provider_id`, but perhaps I am just reading this wrong 😄

Reply:
Yeah was expecting the same but refinement function should test the expected condition, i.e. return falsy to signal failure. So the condition is a negation of your condition.

Reply:
TIL, interesting design choice 😄